index.bs

```diff
@@ -134,6 +140,7 @@ extensible.
 
 <script>
   let pipWindow;
+  const pipButton = document.getElementById('pipButton');
```

Comment:
Shall we add `const video = document.getElementById('video');`as well then?

Reply:
Sorry, I missed that... I just pushed a fix, please take a look!